apps/ui/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "pcgl-daco-ui",
+	"private": true,
+	"version": "0.0.0",
+	"type": "module",
+	"scripts": {
+		"dev": "vite",
+		"build": "tsc -b && vite build",
```

Comment:
I think you can just have tsc only typecheck (not build) and leave the building to Vite - afaik it works on ts files